content/registration_pages/register.en.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Register EN
+date: 2024-02-22T05:58:03.337Z
+description: Register EN
+---
+
+{{% registration_form year="2024" quarter="Autumn"  %}}
```

Comment:
**Annotation**:
The `registration_form` and `reregistration_form` shortcodes accept `year` and `quarter` from the hugo template. This should allow the teachers to change the year and quarter themselves once I build this shortcode into the CMS.